### DISCUSSION TITLE
Clarification of usage behaviour

### DISCUSSION THREAD

**Discussion by hafiidz** [sapphi-red/vite-plugin-static-copy#29] [Q&A] [answered]
I am using electron-vite package boilerplate and I need to find a way to add python as child process. There is an issue in getting some files copied over to the "dist" folder and in [PR#225] (https://github.com/electron-vite/electron-vite-vue/issues/255), this plugin is recommended.

The following is my setup:
![image](https://user-images.githubusercontent.com/3688500/190837415-f692eb95-ace2-4641-9232-6b9b0c4ed65b.png)

How do I make sure a `py` folder is copied over to the `dist` folder ya? Both during dev and build.

**Comment by sapphi-red** [comment, marked as answer]
This plugin does not copy files during dev.
https://github.com/sapphi-red/vite-plugin-static-copy#differences-with-rollup-plugin-copy

If you want to do that, you should use `rollup-plugin-copy`.

**Reply by hafiidz** [reply]
Sorry forgot to update the status. Manage to finally wrap my head around this behaviour earlier today. Thanks for the great work.

**Reply by troncoso** [reply]
@sapphi-red Sorry if I still don't understand, but the README says:

> rollup-plugin-copy for vite with dev server support.

What does that mean if it does not copy files during dev?

**Reply by sapphi-red** [reply]
@troncoso Files are not copied, but they are served directly from dev server.
